test/test_profiler.py

```diff
@@ -1282,6 +1282,45 @@ def test_utils_compute_queue_depth_when_no_cuda_events(self):
         basic_evaluation = _utils.BasicEvaluation(prof.profiler)
         self.assertFalse(basic_evaluation.compute_queue_depth())
 
+    @unittest.skipIf(not torch.cuda.is_available(), "CUDA is required")
```

Comment:
Since we're only testing the ops (CPU side) it's probably to drop cuda so we can test more broadly.